Firestore/Swift/Source/Codable/DocumentID.swift

```diff
@@ -105,12 +106,30 @@ public struct DocumentID<Value: DocumentIDWrappable & Codable>:
   private var value: Value? = nil
 
   public init(wrappedValue value: Value?) {
+    if let value = value {
+      logWarning()
+    }
     self.value = value
   }
 
-  public internal(set) var wrappedValue: Value? {
+  public var wrappedValue: Value? {
     get { value }
-    set { value = newValue }
+    set {
+      logWarning()
+      value = newValue
+    }
+  }
+
+  private func logWarning() {
+    FIRLogDebugSwift(
```

Comment:
At a closer look, we need a FIRLogWarningSwift that calls FIRLogWarning similar to the FIRLogDebug pattern

Reply:
PTAL: 92b9840

Reply:
Pushing another commit to fix a build failure. Locally running the actions to speed up process

Reply:
I'm fairly confident that 0ce4630 will get this to green. Will merge on green.

Reply:
Running workflows locally